.travis.yml

```diff
@@ -0,0 +1,8 @@
+node_js:
+  - "6.11.0"
+
+install:
+  - npm install -g https://github.com/INCF/bids-validator.git#0.25.7
+
+script:
+  - for i in $(ls -d */); do bids-validator ${i%%/} --ignoreNiftiHeaders; done
```

Comment:
I'm not 100% sure, but I'm afraid that exit code of a for loop is the same as the last command in it - this means things failing before last test will be silenced in Travis. See https://stackoverflow.com/questions/14059342/how-to-get-the-exit-status-a-loop-in-bash